packages/amplify-schema-validator/import-check.ts

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs';
+import path from 'path';
+
+const srcDir = path.join(__dirname, 'src');
+const indexPath = path.join(srcDir, 'index.ts');
+const validatorList = fs.readdirSync(path.join(srcDir, 'validators'));
+const indexFileContents = fs.readFileSync(indexPath);
```

Comment:
Rather than this runtime import system which obfuscates dependencies, I'd prefer to explicitly import the validations we're running and losing this check.

Reply:
It's intended as a sort of awkward lint rule - we don't want to contributors to forget to actually import and run their validator. But I'm good with trusting code reviewers to catch that.

Reply:
Yeah, in my mind tests should probably cover 'did my new test catch anything'.

Reply:
Just a warning, if they write their tests to only run their own validator (and not the global export) the tests will not notice this.